include/WCSimPrimaryGeneratorAction.hh

```diff
@@ -67,12 +84,13 @@ private:
 
   // These go with jhfNtuple
   G4int mode;
-  G4int vtxvol;
-  G4ThreeVector vtx;
+  G4int nvtxs;
+  G4int vtxsvol[900];
```

Comment:
I agree with that. I left it as a hard-coded constant because that's the 
way it was before. If everyone is happy I can move into a static const. 
Or should that go as a separate pull request beacause it's an unrelated 
business?

On 06/08/2016 02:07 PM, tdealtry wrote:
>
> In include/WCSimPrimaryGeneratorAction.hh 
> <https://github.com/WCSim/WCSim/pull/171#discussion_r66251064>:
>
> > @@ -67,12 +84,13 @@ private:
> >
> >    // These go with jhfNtuple
> >    G4int mode;
> > -  G4int vtxvol;
> > -  G4ThreeVector vtx;
> > +  G4int nvtxs;
> > +  G4int vtxsvol[900];
>
> Can the 900 value be taken from a new constant? This applies to a few 
> files (e.g. jhfNtuple.h). It'll make it easier to update if/when 
> overflows happen. Of course, vectors are an alternative way to do things
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/WCSim/WCSim/pull/171/files/03e961238070e689bb80cde8c5494fa63a145f65#r66251064>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANMXw36yi2ZvjvhWOF66VdnkMhFnpG41ks5qJr6EgaJpZM4Iw18f>.
>

-- 

_____________________________________________________________
Federico Nova         office: R1 2.89
phone: 01235 44 6265
email:  federico.nova@stfc.ac.uk
_____________________________________________________________
Particle Physics Department            Rutherford Appleton Laboratory
Harwell, Didcot, Oxford, OX11 0QX, UK
_____________________________________________________________